signicat/PublicKeys/0.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<IssuerPublicKey xmlns="http://www.zurich.ibm.com/security/idemix">
+   <Counter>0</Counter>
+   <ExpiryDate>1738581417</ExpiryDate>
```

Comment:
@michielmayer, missed this during my previous review, but I noticed the timestamp at `ExpiryDate` is set to 2 years from now. This should be 1 year.

You can easily change this value by the way, no need to generate a new keypair

Reply:
ok, I changed the expiry date to 1 year from now